integreat_cms/cms/templates/pages/page_sbs.html

```diff
@@ -14,7 +14,7 @@ <h1 class="heading">
             </h1>
             <div class="flex flex-wrap grow justify-between gap-2 mb-4">
                 <a href="{% url 'edit_page' page_id=source_page_translation.page.id region_slug=request.region.slug language_slug=target_language.slug %}"
-                   class="bg-gray-400 hover:bg-gray-500 text-center cursor-pointer text-white font-bold py-3 px-4 rounded">
+                   class="btn btn-outline hover:bg-gray-500 text-center cursor-pointer text-white font-bold py-3 px-4 rounded">
```

Comment:
```suggestion
                   class="btn btn-outline">
```
Ah, I think all those classes are not required anymore, since all the styling is already applied by the `btn` classes...